templates/crane.conf.erb

```diff
@@ -5,5 +5,5 @@
 [general]
 debug: false
 #data_dir_polling_interval: 60
-data_dir: /var/www/pub/docker/app
+data_dir: /var/lib/pulp/published/docker/v2/app/
```

Comment:
One more thing - Crane is OK with the app files not being in data_dir itself, it will os.walk() and find subdirs. Now one thing I haven't tried that *might* work is to point it at /var/lib/pulp/published/docker. That could fail because I'm not sure Crane is smart enough not to try to load the other Docker files that aren't meant for it. However, not having tried it maybe it will work?

Anyways, Crane definitely works with a folder tree that has the app files. Crane might work if there are non-app files in that same tree but this is untested. An RFE for the latter if it doesn't work might be reasonable.